test/test_transcript_support_level.py

```diff
@@ -0,0 +1,22 @@
+"""
+Tests for methods which return collections of transcript IDs that aren't
+converting from some type of name or ID.
+"""
+from __future__ import absolute_import
+
+from nose.tools import eq_
+
+from pyensembl import cached_release
+
+def test_transcript_support_level():
+    ensembl93 = cached_release(93)
+    transcript = ensembl93.transcripts_by_name("DDX11L1-202")[0]
+    eq_(transcript.support_level, "1")
```

Comment:
Are the support levels always strings? Do they ever contain anything other than "1" ... "5"?

Reply:
"1" through "5", or "NA". Because of the latter, hesitated to convert to int.